src/core/utils.js

```diff
@@ -8,3 +10,11 @@ export function ngettext(singular, plural, n) {
   }
   return plural;
 }
+
+export function camelCaseProps(obj) {
```

Comment:
The [redux example](https://github.com/reactjs/redux/blob/fac6443c826636d68b044b715c71de2759919844/examples/real-world/middleware/api.js#L2) uses a package called [humps](https://www.npmjs.com/package/humps) that ships with this feature. It also does this in api.js which I think would be better.